server/lomas_server/dp_queries/dp_libraries/opendp.py

```diff
@@ -201,7 +200,7 @@ def get_output_measure(opendp_pipe: dp.Measurement) -> str:
     output_measure = opendp_pipe.output_measure
 
     if not isinstance(output_type, str):
-        if output_type.origin in ["SMDCurve", "Tuple"]:
+        if output_type.origin in ["SMDCurve", "Tuple"]:  # TODO constant.
```

Comment:
Could you add a ticket and the number next to the TODO, or remove the TODO ? I think otherwise we just forget them and they are never done.